ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -84,6 +99,27 @@ message CreatedEvent {
   google.protobuf.StringValue agreement_text = 6;
 }
 
+// View of a create event matched by an interface filter.
+message InterfaceView {
+
+  // The interface implemented by the matched event.
+  // Required
+  Identifier interface_id = 1;
+
+  // Whether the view was successfully computed, and if not,
+  // the reason for the error. The error is reported using the same rules
+  // for error codes and messages as the errors returned for API requests.
+  // Required
+  google.rpc.Status view_status = 2;
```

Comment:
For my understanding, why can the view computation fail?

Reply:
It's an arbitrary Daml function, so e.g., a division by zero or a call to `error "Thou shall not compute :dragon: !11!!1"